client/src/api/block_builder/input_selection/manual.rs

```diff
@@ -46,24 +46,25 @@ impl<'a> ClientBlockBuilder<'a> {
                 let output = Output::try_from_dto(&output_response.output, token_supply)?;
 
                 if !output_response.metadata.is_spent {
-                    let (_output_amount, output_address) = ClientBlockBuilder::get_output_amount_and_address(
-                        &output,
-                        governance_transition.clone(),
+                    let alias_transition = is_alias_transition_internal(&output, *input.output_id(), &self.outputs);
```

Comment:
Maybe do .0 here instead of mapping later, pretty sure that's compiled away anyway but it looks nicer

Reply:
But here we also would need to map?

Reply:
Ah yeah I'm being dumb, don't mind me 🤦🏻